pyERGM/metrics.py

```diff
@@ -502,6 +502,14 @@ def _numba_calculate_for_sample(networks_sample: np.ndarray, edge_weights: np.nd
 
         return res
 
+    def calculate_mple_regressors(self, observed_network: np.ndarray, edges_indices_lims: tuple[int]):
+        # edge_weights shape is (num_weight_mats, n_nodes, n_nodes)
+        num_nodes = len(self.exogenous_attr)
+        num_weight_mats = self.edge_weights.shape[0]
+        # remove diagonal and reshape to the desired format
+        Xs = self.edge_weights[:, np.eye(num_nodes) == 0].reshape(num_nodes**2 - num_nodes, num_weight_mats)
```

Comment:
I think transpose is better than reshape (both in terms of readability, and in terms of efficiency)